packages/documentation-site/patternfly-docs/content/design-guidelines/usage-and-behavior/usage-and-behavior.md

```diff
@@ -3,119 +3,138 @@ id: Usage and behavior
+|You want to optimize numeric data entry for direct manipulation by touch or using the mouse over the keyboard.|Number input or slider|
+|It’s useful for users to visualize where numeric input falls within the range of possible values.| Slider |
 
+## Selecting between options on a form
+There are a few components that you can use to allow users to select form options. Depending on the nature of options being presented, we recommend using the **[checkbox](/components/forms/checkbox)**, **[radio](/components/forms/radio)**, or **[switch](/components/switch)** components. 
 
-**[1]** Switches are sometimes used in place of checkboxes in this use case. The advantage of a switch is that it is more mobile friendly as it provides a larger touch target than the standard checkbox.
-
-**[2]** Checkboxes are often used to enable or disable (turn on or off) a feature in software. They provide a concise way to expose on/off settings while being less explicit than a switch.
+- Checkboxes allow users to select one or more items from a list of options. 
 
-**[3]** You could use two radio buttons to support choosing between on/off or enabled/disabled options, however this approach uses more space and is not recommended unless it’s important to make both labeled options visible at the same time.
+- Radio buttons allow users to select from a set of mutually exclusive options.
 
-## Displaying progress
-It’s important to display progress as a method of providing user feedback for operations that will take more than a few seconds. Use the [progress](/components/progress) component to provide feedback on the percentage of completion for a process or task. Use the [spinner](/components/spinner) and [skeleton](/components/skeleton) components to just indicate activity while the user is waiting for an action to complete.
+- Switches indicate the state of a binary setting or object (such as on/off, enabled/disabled).
 
-| Use case | [Progress](/components/progress) | [Spinner](/components/spinner) | [Skeleton](/components/skeleton) |
-|------------------------ | :---: | :-------: | :---------------: |
-| The user is waiting for a process to complete, but the time left until completion is not known.|⛔|✔|⛔|
-|The user is waiting for a process to complete, but the time left until completion is known.|✔|✔ [1]|⛔|
-|The user is progressing through a step-by-step task and you want to reflect where they are in the process.|✔|⛔|⛔|
-|The user is waiting for a page to load.|✔ [2]|✔ [3]|✔|
+Your use case dictates which of these components you should use:
 
+| **Use case** | **Recommended component** |
+|------------------------ | -----------|
+|The user wants to select one or more items from a list of items.| Checkbox or switch <br /><br /> **Note:** Switches are sometimes used in place of checkboxes because they provide a larger touch target than a checkbox, which improves the mobile experience.
+|
+|The user wants to select from a set of mutually exclusive options.| Radio |
```

Comment:
```suggestion
|The user wants to select one or more items from a list of items. | Checkbox or switch <br /><br /> **Note:** Switches are sometimes used in place of checkboxes because they provide a larger touch target than a checkbox, which improves the mobile experience. |
| The user wants to select from a set of mutually exclusive options.| Radio |
```